### PR TITLE
Fe jy news task7

### PR DESCRIPTION
**Pull request by jamesyuill**

### CODE REVIEW

File: src/components/VotesComp.jsx [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+      .then((res) => {
+        console.log(res); //<--i will remove, but thought i should leave in so you could see it working
+      })
+      .catch((err) => {
+        setVoteClicks((curr) => {
+          return curr - num;
+        });
+      });
+  }
+
```
Comment (NRMSMITH) [inline src/components/VotesComp.jsx:27]: I should be able to see if the vote has not worked - via something appearing - whether that's a modal saying 'sorry vote unsuccessful' or something along those lines!

Comment (NRMSMITH) [reply src/components/VotesComp.jsx:27]: (Also, I really like the greying out of the vote that's already been clicked!

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
Just a quick one about notifying when the call hasn't been successful!

File: src/components/VotesComp.jsx (hunk 1 above)
Comment (jamesyuill) [reply src/components/VotesComp.jsx:27]: oh i didn't include the setstate function in the catch block. i had set something up if the response from the server was an error, but linked the two!

**Comment by NRMSMITH** [comment]
Approved! 👍

**Pull request merged**